functions-and-operators/json-functions.md

```diff
@@ -132,4 +133,6 @@ TiDB supports most of the JSON functions that shipped with the GA release of MyS
 
 [json_pretty]: https://dev.mysql.com/doc/refman/5.7/en/json-utility-functions.html#function_json-pretty
 
+[json_storage_free]: https://dev.mysql.com/doc/refman/8.0/en/json-utility-functions.html#function_json-storage-free
```

Comment:
I think this link is fine as this function was introduced in MySQL 8.0 and isn't in the MySQL 5.7 docs. Not sure if we should explicitly list this as MySQL 8.0 compatibility or not. Also might be good to consider updating all links to MySQL 8.0 at some point, but that's not for this PR

Reply:
> Not sure if we should explicitly list this as MySQL 8.0 compatibility or not.

@qiancai PTAL. I think it would be better to tag as MySQL 8.0 feature, but I don't know whether other docs have similar problem / solution? :thinking: .

> Also might be good to consider updating all links to MySQL 8.0 at some point, but that's not for this PR

As I know, the support for MySQL 8.0 JSON features are still ongoing, and some references can be updated while some are not (e.g. I am working on supporting 8.0 json path syntax this month).

Reply:
Because L9 in this document mentions that "TiDB supports most of the JSON functions that shipped with the GA release of MySQL 5.7.", users would assume that the functions mentioned in this doc are for v5.7 by default. In this case, do you think we can highlight v8.0 as follows?
![image](https://user-images.githubusercontent.com/79440533/198272229-1469a759-b1dd-468e-978b-c99fc08c78f2.png)